CONTRIBUTING.md

```diff
@@ -335,7 +335,28 @@ since the latest release over the hotfixed `master`.
 This would mean, however, that the commits referenced from PRs previously merged to `develop` will no longer be part of `develop`'s history,
 which is rather unacceptable.
 
+### Plugin signing
 
+The valid non-expired key pair together with the password required for the plugin signing should be present in the CI environment.
+If they are absent, please take a look at the [plugin signing in IntelliJ](https://plugins.jetbrains.com/docs/intellij/plugin-signing.html#signing-methods) for updated instructions on how to do it.
+Currently, you would need to first generate the private key with:
+```shell
+openssl genpkey\
+  -aes-256-cbc\
+  -algorithm RSA\
+  -out private.pem\
+  -pkeyopt rsa_keygen_bits:4096
+```
+Then, use that for generating the certificate chain by running:
+```shell
+openssl req\
+  -key private.pem\
+  -new\
+  -x509\
+  -days 365\
+  -out chain.crt
+```
+Please note that you would have to copy the contents of `private.pem` and `chain.crt` as well as the password to the private key, in the corresponding environment variables; in order for the Gradle IntelliJ Plugin to pick them up and use them for the plugin signing task, before publishing to the Marketplace.
```

Comment:
Actually, I was wrong about local runs. There might indeed be a case where a developer wants to sign the plugin locally with the freshly-generated keys... right before they decide to set the new signing key in the CI, to test locally if everything works. That's actually my own case right now 😅 So if you could include the instructions of `export`ing the env vars needed for signing locally.